x/concentrated-liquidity/types/constants.go

```diff
@@ -8,7 +8,13 @@ import (
 
 const (
 	// Precomputed values for min and max tick
-	MinTick, MaxTick              int64 = -162000000, 342000000
+	MinInitializedTick, MaxTick int64 = -162000000, 342000000
+	// If we consume all liquidity and cross the min initialized tick,
+	// our current tick will equal to MinInitializedTick - 1 with zero liquidity.
+	// However, note that this tick cannot be crossed. If current tick
+	// equals to this tick, it is only possible to swap in the right (one for zero)
+	// direction.
+	MinCurrentTick                int64 = MinInitializedTick - 1
```

Comment:
What's the reason we don't have this for max tick?

Reply:
I think the reasoning is the lowerTick is inclusive whereas upper is not

Reply:
Correct. Clarified with a comment: [3822af3](https://github.com/osmosis-labs/osmosis/pull/5581/commits/3822af30062d3a717ede403a7b33ac7a9c3c5c05)